extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -692,6 +734,42 @@ void ConsumeWindowsEventLog::putEventRenderFlowFileToSession(const EventRender&
     session.getProvenanceReporter()->receive(flowFile, provenanceUri_, getUUIDStr(), "Consume windows event logs", 0);
     session.transfer(flowFile, Success);
   }
+
+  if (output_.json.raw) {
+    auto flowFile = session.create();
+    logger_->log_trace("Writing rendered raw JSON to a flow file");
+
+    session.write(flowFile, &WriteCallback(eventRender.json.raw));
+    session.putAttribute(flowFile, core::SpecialFlowAttribute::MIME_TYPE, "application/json");
+    session.putAttribute(flowFile, "Timezone name", timezone_name_);
+    session.putAttribute(flowFile, "Timezone offset", timezone_offset_);
+    session.getProvenanceReporter()->receive(flowFile, provenanceUri_, getUUIDStr(), "Consume windows event logs", 0);
+    session.transfer(flowFile, Success);
```

Comment:
These lines seem to be the same for all json output formats. We could a single branch for all json output formats and only distinguish the `session.write` calls by json format type.

Reply:
factored out common code into a lambda